cookbooks/fb_users/README.md

```diff
@@ -170,6 +171,27 @@ automatically cleaned up.
 
 Also see `initialize_group` helper below.
 
+The `notifies` is a hash that looks like this:
+
+```ruby
+node.default['fb_users']['users']['zerocool'] = {
+  'restart foo if uid changes' => {
```

Comment:
shouldn't this be `notifies` based on `info['notifies']` on line 55 in the resource?
```suggestion
  'notifies' => {
    'restart foo if uid changes' => {
```